apps/minifront/src/components/shared/input-token.tsx

```diff
@@ -47,13 +49,18 @@ export default function InputToken({
           variant='transparent'
           placeholder={placeholder}
           className={cn(
-            'md:h-8 xl:h-10 md:w-[calc(100%-80px)] xl:w-[calc(100%-160px)] md:text-xl  xl:text-3xl font-bold leading-10',
+            'md:h-8 xl:h-10 md:w-[calc(100%-80px)] xl:w-[calc(100%-160px)] md:text-xl xl:text-3xl font-bold leading-10',
             inputClassName,
           )}
           value={value}
           onChange={e => onInputChange(e.target.value)}
         />
-        <BalanceSelector value={selection} onChange={setSelection} balances={balances} />
+        <BalanceSelector
```

Comment:
general question, can the loading somehow get stuck and never properly render the async UI component?

Reply:
Hm... I think it'd eventually time out. We don't currently have error handling, unfortunately — I'd like to handle that in the future. I'll bring it up with Sam during our next UI sync.